<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetPortSettings</class>
 <widget class="QWidget" name="NetPortSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>227</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>339</width>
    <height>227</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>339</width>
    <height>227</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>网口设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/set.ico</normaloff>:/images/set.ico</iconset>
  </property>
  <widget class="QLineEdit" name="LocalPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>8080</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ApplyButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="OsideIpLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>远程地址：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LocalIp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="OppositeIp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="OppositePort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>6</y>
     <width>120</width>
     <height>125</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>122</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>122</height>
    </size>
   </property>
   <property name="title">
    <string>网络端口选择</string>
   </property>
   <widget class="QRadioButton" name="UDPButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TCPCButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP Client</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="TCPSButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP Server</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="LocalPortLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>本机端口：</string>
   </property>
  </widget>
  <widget class="QLabel" name="OsidePortLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>远程端口：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="AddrgroupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="LocalIpLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>本机地址：</string>
    </property>
   </widget>
  </widget>
  <zorder>AddrgroupBox</zorder>
  <zorder>LocalPort</zorder>
  <zorder>ApplyButton</zorder>
  <zorder>OsideIpLabel</zorder>
  <zorder>LocalIp</zorder>
  <zorder>OppositeIp</zorder>
  <zorder>OppositePort</zorder>
  <zorder>groupBox</zorder>
  <zorder>LocalPortLabel</zorder>
  <zorder>OsidePortLabel</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
